### PR TITLE
Xendit JS for React Native

### PR DESCRIPTION
**Pull request by PatrickBenz09**
## Problem:
* Need to have new xendit.js as a compatible plugin for react-native

## Solution:
* create it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+To deploy the example files manually, run:
+```
+grunt deploy:examples
+```
+
+To deploy the xendit.js library manually, run:
+```
+grunt deploy:xenditjs --build-version=BUILD_VERSION_HERE
+```
+In the above command, build version must be a valid version (e.g. `1.0.0`) and a build for that version must exist.
```
Comment (0505gonzalez) [inline README.md]: These instructions don't belong in the public repo

File: build/xendit-0.1.0.js [hunk 2]
```diff
@@ -0,0 +1,435 @@
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+'use strict';
+
```
Comment (0505gonzalez) [inline build/xendit-0.1.0.js]: No need to publish builds since this is a node module

File: package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/xendit/xendit.js.git"
+  },
+  "author": "0505gonzalez",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/xendit/xendit.js/issues"
```
Comment (0505gonzalez) [inline package.json]: need to change repo urls

File: package.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "xendit.js",
+  "version": "0.6.1",
+  "description": "JS client library for tokenizing credit cards",
+  "main": "index.js",
```
Comment (0505gonzalez) [inline package.json]: There's no index.js so publishing the node module won't work

File: src/card.js [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+if (!window.btoa) {
+    window.btoa = require('base-64').encode;
+}
+
+function Card (Xendit) {
+    this._xendit = Xendit;
+
+    Xendit._device_fingerprint_id = DeviceFingerprintUtil.createDeviceFingerprintID();
+
+    DeviceFingerprintUtil.mountDeviceFingerprintHTML(Xendit._device_fingerprint_id);
```
Comment (afik) [inline src/card.js]: we dont need this and the line above, please remove

File: src/utils/device_fingerprint_util.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    //             '<iframe style="width: 100px; height: 100px; border: 0; position: absolute; top: -5000px;"" src="' + iframeURL + '"></iframe>' +
+    //         '</noscript>';
+    //
+    //     window.removeEventListener('load', onload, false);
+    //
+    //     document.getElementsByTagName('body')[0].appendChild(containerEl);
+    // }, false);
+};
+
+module.exports = DeviceFingerprintUtil;
```
Comment (afik) [inline src/utils/device_fingerprint_util.js]: remove this file

File: src/utils/window_message_util.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    //         if (message.id === messageId) {
+    //             messageHandler(message);
+    //         }
+    //     } catch (e) {
+    //         //
+    //     }
+    // }, false);
+};
+
+module.exports = WindowMessageUtil;
```
Comment (afik) [inline src/utils/window_message_util.js]: remove this file

File: src/card.js [hunk 8]
```diff
@@ -0,0 +1,233 @@
+'use strict';
+
+var RequestUtil = require('./utils/request_util');
+var CreditCardUtil = require('./utils/credit_card_util');
+var WindowMessageUtil = require('./utils/window_message_util');
+var DeviceFingerprintUtil = require('./utils/device_fingerprint_util');
```
Comment (afik) [inline src/card.js]: remove this and the line above

**Comment by afik** [comment]
LGTM

**Pull request merged**